inference-engine/tests/functional/ir_v7_reader/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME ieIRV7Tests)
+
+addIeTargetTest(
```

Comment:
IE Core loads readers in the runtime, we cannot avoid loading of v7 readers if we just create a new target.